src/main/java/moze_intel/projecte/utils/EMCHelper.java

```diff
@@ -307,6 +308,18 @@ public static String getEmcSellString(ItemStack stack, int stackSize)
 		return " (" + Constants.EMC_FORMATTER.format((emc * stackSize)) + ")";
 	}
 
+	public static String getEmcSellStringBig(ItemStack stack, int stackSize)
+	{
+		if (EMCMapper.covalenceLoss == 1.0)
+		{
+			return " ";
+		}
+
+		BigInteger emc = BigInteger.valueOf(EMCHelper.getEmcSellValue(stack));
+
+		return " (" + Constants.EMC_FORMATTER.format(emc.multiply(BigInteger.valueOf(stackSize))) + ")";
+	}
+
```

Comment:
don't see the point of an extra method tbh, just make the one above use bigintegers too.

Reply:
Done